react-hot-button-component/my-app/src/HotButton.js

```diff
@@ -0,0 +1,29 @@
+import { useState } from 'react';
+
+export default function HeatButton() {
```

Comment:
The file is named 'HotButton' but the component is 'HeatButton'. It works, but it's nice to keep them consistent.